sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -51,12 +54,78 @@ def test_generate_policy():
         topics_subscribe_allowed = profile.find(path='topics[@subscribe="ALLOW"]')
         assert topics_subscribe_allowed is not None
 
-        # Verify that the allowed publications include topic_pub and not topic_sub
+        # Verify that the allowed publications include topic_pub and topic_both, but not
+        # topic_sub
         topics = topics_publish_allowed.findall('topic')
         assert len([t for t in topics if t.text == 'topic_pub']) == 1
+        # FIXME: Why the leading slash?
+        assert len([t for t in topics if t.text == '/topic/both']) == 1
         assert len([t for t in topics if t.text == 'topic_sub']) == 0
 
-        # Verify that the allowed subscriptions include topic_sub and not topic_pub
+        # Verify that the allowed subscriptions include topic_sub and topic_both, but not
+        # topic_pub
+        topics = topics_subscribe_allowed.findall('topic')
+        assert len([t for t in topics if t.text == 'topic_sub']) == 1
+        # FIXME: Why the leading slash?
```

Comment:
This doesn't seem quite right, but this is how the code worked before this change, so I kept it.